### ISSUE TITLE
Allow SSH options on command line or csshrc

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brockgr/csshx#8]
```
Support:

   csshX -o "-x -o ConnectTimeout=10"

csshrc:

   ssh_args = "-x -o ConnectTimeout=10"
```

Original issue reported on code.google.com by `gavin.brock` on 21 Apr 2009 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
Now in SVN version.. Skipped "-o" it's only "--ssh_args" for now
```

Original comment by `gavin.brock` on 22 Apr 2009 at 11:06
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released in 0.61
```

Original comment by `gavin.brock` on 30 Apr 2009 at 2:43
- Changed state: **Done**
